recipe/meta.yaml

```diff
@@ -12,8 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  # skip: True  # [py<36]
-  # noarch: python
```

Comment:
Is this a rebuild after PR #4 was merged? What happened to the packages from that merge?

Reply:
Do you know why conda-forge decided to pin to python >=3.6? That doesn't seem to be a restriction imposed by the upstream project.

Reply:
the old packages are still in the prefect folder but have not been pushed to default just yet we can delete them before we merge

Reply:
the artifacts from the PR has been deleted